_common/qiskit/benchmark_runner.py

```diff
@@ -51,9 +51,9 @@
     parser.add_argument("--hub", default="ibm-q", help="Computing group hub.", type=str)
     parser.add_argument("--group", default="open", help="Group status", type=str)
     parser.add_argument("--project", default="main", help="Project", type=str)
-    parser.add_argument("--provider_module_name", default= None , help="provider_module_name", type= str)
-    parser.add_argument("--provider_class_name", default= None , help="provider_class_name", type= str)
-    parser.add_argument("--noise_model", default= None , help="Noise Model", type= str)
+    parser.add_argument("--provider_module_name", default=None, help="provider_module_name", type= str)
```

Comment:
The `help` text should probably be different than just the name of the argument; it should be a description of what this argument is and how to specify it.